Resources/Textures/Objects/Fun/spraycans.rsi/meta.json

```diff
@@ -70,6 +79,22 @@
     {
       "name": "equipped-BELT",
       "directions": 4
+    },
+    {
+      "name": "clown-inhand-right",
+      "directions": 4
+    },
+    {
+      "name": "clown-inhand-left",
+      "directions": 4
+    },
+    {
+      "name": "spray-inhand-right",
+      "directions": 4
+    },
+    {
+      "name": "spray-inhand-left",
+      "directions": 4
     }
   ]
 }
```

Comment:
Also its bad working with energy weapon

https://github.com/space-wizards/space-station-14/assets/96445749/f33b7f08-59d7-45b6-b4e4-31889bc199a6

Reply:
Not much that can be done here except exclude energy weapons from being paintable with a component.